test/k8sT/manifests/graceful-termination.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: graceful-term-svc
+spec:
+  ports:
+    - port: 8081
+  selector:
+    app: graceful-term-server
+---
+apiVersion: v1
```

Comment:
It would make reviewing easier if you could write a few comments saying what the server and the client does, and what output/behavior is used for testing. Otherwise, a reviewer needs to dive into the implementations of both.

Reply:
There is a detailed description of the test apps in the commit description, and also in the e2e test inline. I don't see any description in other test yamls. I'll see what I can do more.